<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoButton</class>
 <widget class="QWidget" name="TwoButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>477</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_twobt_title">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_twobt_lb1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_twobt_edit1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_twobt_ok">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/cancel.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_twobt_return">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirm.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_twobt_lb2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/wifibg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_twobt_title</zorder>
  <zorder>label_twobt_lb1</zorder>
  <zorder>lEdit_twobt_edit1</zorder>
  <zorder>pButton_twobt_ok</zorder>
  <zorder>pButton_twobt_return</zorder>
  <zorder>label_twobt_lb2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
